test/kata3.numberToReversedDigits.test.js

```diff
@@ -1,5 +1,7 @@
 const { numberToReversedDigits } = require("../src");
 
 describe("numberToReversedDigits", () => {
-  test("returns a reversed array of the number's digits", () => {});
-});
\ No newline at end of file
+	test("returns a reversed array of the number's digits", () => {
```

Comment:
Would be good to add another couple of tests in here, could check for different lengths of array for example?